### PR TITLE
Update versions used in plugins CI

### PR DESCRIPTION
**Pull request by andsel**
**Merge after DRA build of 8.8.1**

Updated Logstash versions used in CI

### CODE REVIEW

File: ci/logstash_releases.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
     "5.x": "5.6.16",
     "6.x": "6.8.23",
     "7.x": "7.17.10",
-    "8.x": "8.7.1"
+    "8.x": "8.8.0"
   },
   "snapshots": {
     "7.x": "7.17.11-SNAPSHOT",
-    "8.x": "8.8.0-SNAPSHOT"
+    "8.x": "8.8.1-SNAPSHOT"
```
Comment (jsvd) [inline ci/logstash_releases.json:10]: Let's have plugins test against the next minor instead:

```suggestion
    "8.x": "8.9.0-SNAPSHOT"
```

**Review by jsvd — CHANGES_REQUESTED** [review]

**Review by jsvd — APPROVED** [review]
LGTM

**Comment by andsel** [comment]
Jenkins test this please

**Comment by andsel** [comment]
Jenkins test this please

**Pull request merged**